auth_manager.go

```diff
@@ -147,22 +139,9 @@ func (b *DefaultSessionManager) UpdateSession(keyName string, session *user.Sess
 		err = b.store.SetKey(keyName, string(v), resetTTLTo)
 	}
 
-	if err == nil {
-		b.renewSessionState(keyName, session)
-	}
-
 	return err
 }
 
-func (b *DefaultSessionManager) renewSessionState(keyName string, session *user.SessionState) {
-	// we have new session state so renew first-seen hash to prevent
-	session.SetFirstSeenHash()
-	// delete it from session cache to have it re-populated next time
-	if !b.disableCacheSessionState {
```

Comment:
looks like we don't need this field any more https://github.com/TykTechnologies/tyk/pull/1770/files#diff-48bee4488d3d533801a48f81a4de665fR49

and initializing part here: https://github.com/TykTechnologies/tyk/pull/1770/files#diff-48bee4488d3d533801a48f81a4de665fR88

Reply:
removed